File: Modules/SMES/Esop/SIE.Web.ESop/EngDocuments/Commands/EngDocUseTypeCommand.cs
using SIE.Web.Command;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SIE.Web.ESop.EngDocuments.Commands
{
    /// <summary>
    /// 使用类型维护
    /// </summary>
    public class EngDocUseTypeCommand : ViewCommand
    {
        /// <summary>
        /// 使用类型维护
        /// </summary>
        /// <param name="args"></param>
        /// <param name="scope"></param>
        /// <returns></returns>
        protected override object Excute(ViewArgs args, string scope)
        {
            return true;
        }
    }
}

File: Modules/SMES/Esop/SIE.Web.ESop/EngDocuments/Commands/EngDocUseTypeCommand.js
SIE.defineCommand('SIE.Web.ESop.EngDocuments.Commands.EngDocUseTypeCommand', {
    meta: { text: "使用类型维护", group: "edit", iconCls: "icon-DocumentLabel icon-blue" },
    execute: function (view, source) {
        CRT.Workbench.addPage({
            entityType: 'SIE.ESop.EngDocuments.FileUseDetail',
            title: '工程文件使用类型'.L10N(),
            module: "SIE.ESop.EngDocuments.FileUseDetail,SIE.ESop",
            isAggt:true
        });
    }
});
